<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialogBase</class>
 <widget class="QDialog" name="PreferenceDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>General Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="entrySaveCheckbox">
        <property name="text">
         <string>Re-use previous day's comments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="powerUserCheckbox">
        <property name="text">
         <string>Poweruser view</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="singleClickCheckbox">
        <property name="text">
         <string>Activate account by single click</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showTypeCheckBox">
        <property name="text">
         <string>Show account type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showPSPCheckBox">
        <property name="text">
         <string>Show PSP element number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useDefaultCommentIfUniqueCheckBox">
        <property name="text">
         <string>Automatically use default comment if it is unique</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dragNDropCheckbox">
        <property name="text">
         <string>Activate drag 'n' drop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="persoenlicheKontensummeCheckbox">
        <property name="text">
         <string>Show sum in personal accounts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showSpecialRemunSelector">
        <property name="text">
         <string>Show special remuneration selector</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="defaultCommentsCheckBox">
     <property name="title">
      <string>Default Comments Display Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="radioDefCommBold">
        <property name="text">
         <string>Used default comments in bold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioAvailabeDefCommNotSelectedBold">
        <property name="text">
         <string>Not used default comments in bold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioNoBold">
        <property name="text">
         <string>Don't mark default comments</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Increments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="TextLabel1">
          <property name="text">
           <string>Basic time increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="zeitIncBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="TextLabel1_2">
          <property name="text">
           <string>Fast time increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fastZeitIncBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Fonts</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="customFontCheckBox">
          <property name="text">
           <string>Use custom font</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="customFontSelectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select Font</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="fontPreview">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>40</number>
        </attribute>
        <column>
         <property name="text">
          <string>Accounts</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Active</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Accountable</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Comment</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>All Accounts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personal Accounts</string>
         </property>
         <item>
          <property name="text">
           <string>Department</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>management</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string>0:30</string>
           </property>
           <property name="text">
            <string>0:30</string>
           </property>
           <property name="text">
            <string>talked to boss about workload</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>training</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string>0:00</string>
           </property>
           <property name="text">
            <string>0:00</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Customer 1</string>
          </property>
          <item>
           <property name="text">
            <string>project1</string>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>204</green>
              <blue>102</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>204</green>
              <blue>102</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>204</green>
              <blue>102</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>204</green>
              <blue>102</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>204</green>
              <blue>102</blue>
             </color>
            </brush>
           </property>
           <item>
            <property name="text">
             <string>0</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>2:00</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>1:45</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>web server setup: some work done, lots to do</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>0:15</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>0:15</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>backup: tapes changed</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>0:00</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string>0:00</string>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="background">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>204</green>
               <blue>102</blue>
              </color>
             </brush>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>project2</string>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>128</green>
              <blue>128</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>128</green>
              <blue>128</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string>0:00</string>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>128</green>
              <blue>128</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string>0:00</string>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>128</green>
              <blue>128</blue>
             </color>
            </brush>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>128</green>
              <blue>128</blue>
             </color>
            </brush>
           </property>
          </item>
         </item>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" stretch="0,0,0">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>entrySaveCheckbox</tabstop>
  <tabstop>powerUserCheckbox</tabstop>
  <tabstop>singleClickCheckbox</tabstop>
  <tabstop>showTypeCheckBox</tabstop>
  <tabstop>showPSPCheckBox</tabstop>
  <tabstop>useDefaultCommentIfUniqueCheckBox</tabstop>
  <tabstop>dragNDropCheckbox</tabstop>
  <tabstop>persoenlicheKontensummeCheckbox</tabstop>
  <tabstop>showSpecialRemunSelector</tabstop>
  <tabstop>radioDefCommBold</tabstop>
  <tabstop>radioAvailabeDefCommNotSelectedBold</tabstop>
  <tabstop>radioNoBold</tabstop>
  <tabstop>zeitIncBox</tabstop>
  <tabstop>fastZeitIncBox</tabstop>
  <tabstop>customFontCheckBox</tabstop>
  <tabstop>customFontSelectButton</tabstop>
  <tabstop>fontPreview</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>customFontCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>customFontSelectButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customFontCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontPreview</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
